tuxemon/networking.py

```diff
@@ -24,7 +24,7 @@
 from tuxemon.middleware import Controller, Multiplayer
 from tuxemon.npc import NPC
 from tuxemon.session import local_session
-from tuxemon.states import world
+from tuxemon.states.world import WorldState
```

Comment:
Do we need this change? I think this is causing the python3.10 tests error.

Reply:
No, we don't need it at the moment.